app/views/users/_public_user_data.html.erb

```diff
@@ -0,0 +1,23 @@
+<%- model_class = User -%>
+
+<h4><%=t('.public_data')%></h4><br>
+
+<% [:first_name, :last_name].each do |field| %>
+	<% unless @user[field].blank? %>
+		<p><strong><%= model_class.human_attribute_name(field)%>: </strong><%= @user[field] %></p>
+	<% end %>
+<% end %>
+
+<% unless @user.chair.nil? %>
+	<p><strong><%= model_class.human_attribute_name(:chair)%>: </strong><%= @user.chair.name %></p>
+<% end %>
+
+<% unless @user.projects.blank? %>
+	<p>
+		<strong><%= t('.projects')%>: </strong>
+		<% @user.projects[0..-2].each do |project| %>
```

Comment:
Eigentlich möchte ich hier .join(', ') verwenden aber das geht nicht weil ich die Links joinen muss. ich habe das hier versucht:
@user.projects.map{ |project| link_to(project.title, project)}.join(', ')
aber die links werden in Anfürhunsstriche gesetzt und somit als plain text angezeigt. Vielleicht fällt jemandem noch was besseres ein?